PKGBUILD

```diff
@@ -36,11 +35,10 @@ provides=('chromium')
 conflicts=('chromium')
 source=(https://commondatastorage.googleapis.com/chromium-browser-official/chromium-${_chromium_version}.tar.xz
         chromium-launcher-$_launcher_ver.tar.gz::https://github.com/foutrelis/chromium-launcher/archive/v$_launcher_ver.tar.gz
-        "ungoogled-chromium-${_ungoogled_version}.tar.gz::https://github.com/Eloston/ungoogled-chromium/archive/${_ungoogled_version}.tar.gz"
-        "ungoogled-chromium-archlinux-${_ungoogled_archlinux_version}.tar.gz::https://github.com/ungoogled-software/ungoogled-chromium-archlinux/archive/${_ungoogled_archlinux_version}.tar.gz")
+        "git+https://github.com/ungoogled-software/ungoogled-chromium-archlinux#branch=master"
```

Comment:
So we always checkout the tip of master? Will this break builds if we push changes to `master` that we haven't tested yet? Or is `master` supposed to be always stable?

Also, do we have some way to figure out which commit they ended up building the PKGBUILD with if users come to report bugs to us?

Reply:
Generally, I prefer not to checkout the tip of master, since that runs the risk of breaking builds because of untested changes. I'm hoping to rely on tags instead, just like with `ungoogled-chromium` to avoid that risk.

Reply:
So this line will be changed to use a tag instead?